app.js

```diff
@@ -0,0 +1,116 @@
+var memory = {
+  initial: [],
+  operation: undefined,
+  latter: []
+}
+
+function operationAdder(div) {
+  memory.operation = this.innerHTML
+}
+
+function numberCounter(div) {
+  if( !memory.operation && (this.innerHTML !== 'AC' || '+/-' || '%' || '.' )) {
+      memory.initial.push(this.innerHTML)
+      document.querySelector('.calculator-screen').innerHTML = memory.initial.join('')
+  }else if( memory.operation && (this.innerHTML !== 'AC' || '+/-' || '%' || '.' )) {
+      memory.latter.push(this.innerHTML)
+      document.querySelector('.calculator-screen').innerHTML = memory.latter.join('')
+  }
+}
+
+document.querySelectorAll(".calculator-button").forEach(function(div) {
+  div.addEventListener('click', numberCounter )
+  div.addEventListener('click', function(){
+    if(div.innerHTML === "AC"){
+      memory = {
+        initial: [],
+        operation: undefined ,
+        latter: []
+      }
+    }
+
+    if(memory.initial.length > 0 ){
+      document.querySelector('.clear').innerHTML = 'C'
+    }
+
+    if(div.innerHTML === "C" && !memory.operation ){
+      memory.initial = []
+      document.querySelector('.calculator-screen').innerHTML = 0
+      div.innerHTML = "AC"
+    }
+    if(div.innerHTML === "C" && memory.operation ){
+      memory.latter = []
+      document.querySelector('.calculator-screen').innerHTML = 0
+      div.innerHTML = "AC"
+    }
+
+    if (div.innerHTML === '+/-' && memory.latter.length === 0 ){
+      var value = parseFloat(memory.initial.join(''))
+      memory.initial = (value * -1).toString().split('')
+    }else if (div.innerHTML === '+/-' && memory.latter.length > 0 ){
+      var value = parseFloat(memory.latter.join(''))
+      memory.latter = (value * -1).toString().split('')
+    }
+
+    if (div.innerHTML === '%' && memory.latter.length === 0 ){
+      var value = parseFloat(memory.initial.join(''))
+      memory.initial = (value / 100).toString().split('')
+    }else if (div.innerHTML === '%' && memory.latter.length > 0 ){
+      var value = parseFloat(memory.latter.join(''))
+      memory.latter = (value / 100).toString().split('')
+    }
+
+    if( (div.innerHTML === '.') && (memory.latter.length === 0) && (memory.initial.indexOf('.') === -1 ) ){
+      memory.initial.push('.')
+      document.querySelector('.calculator-screen').innerHTML = memory.initial.join('')
+    }else if( (div.innerHTML === '.') && (memory.latter.length > 0) && (memory.latter.indexOf('.') === -1 ) ){
+      memory.latter.push('.')
+      document.querySelector('.calculator-screen').innerHTML = memory.latter.join('')
+    }
+  })
+})
+
+document.querySelectorAll(".operator").forEach(function(div) {
+  div.addEventListener('click', operationAdder )
+  div.removeEventListener('click', numberCounter )
```

Comment:
Removing these means you didn't properly add `numberCounter` as an event listener.